Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net471;net472</TargetFrameworks>

```

Comment:
@sfc-gh-ext-simba-lf maybe update your PR to the current master, and then prepare also a modification of README.md. We want to write a disclaimar that we tested the library only on particular environments and do not guarantee how it will work with other environements. As soon as its ready, ask @sfc-gh-mheavin for review of the documentation.

Reply:
Will do, for now the PR is updated to master